modules/serverless-list-subs-kn.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+//  * serverless/event_workflows/serverless-subs.adoc
+
+[id="serverless-list-subs-kn_{context}"]
+= Listing subscriptions using the Knative CLI
+
+You can list existing subscriptions on your cluster by using the `kn` CLI.
+
+.Prerequisites
+
+* You have installed the `kn` CLI.
+
+.Procedure
+
+* List subscriptions on your cluster:
++
+[source,terminal]
+----
+$ kn subscription list
+----
++
+.Example output
+[source,terminal]
+----
+NAME            CHANNEL             SUBSCRIBER           REPLY   DEAD LETTER SINK   READY   REASON
+mysubscription   Channel:mychannel   ksvc:event-display                              True
```

Comment:
This looks unaligned (one extra space) even in the rendered doc on https://deploy-preview-30282--osdocs.netlify.app/openshift-enterprise/latest/serverless/event_workflows/serverless-subs.html#serverless-creating-subscriptions-kn_serverless-subs

Reply:
@maschmid can you add it as a suggestion please so I can see exactly where?